### ISSUE TITLE
techdocs-cli serve 404

### ISSUE THREAD

**Issue by ericis** [backstage/backstage#11177]
See: https://github.com/ericis/test-techdocs-cli-serve *a documentation-only (TechDocs) site

## Expected Behavior

Serves "./docs/index.md" without error.

## Actual Behavior

![image](https://user-images.githubusercontent.com/974919/165984706-f8ccdc27-9745-49b8-9dd7-96bcad2332b1.png)

## Steps to Reproduce

Using the latest stable "@techdocs/cli" or attempting to use the nightly release "@techdocs/cli@0.0.0-nightly-20220429024638" both result in a 404 failure.

1. `git clone https://github.com/ericis/test-techdocs-cli-serve`
2. `yarn`
3. `yarn start` (calls `techdocs-cli serve`

## Context

Unable to test a TechDocs site using the official TechDocs CLI.

## Your Environment

- Browser Information: Firefox 99.0.1

- Output of `yarn backstage-cli info`: 

```text
n/a (nothing, because this uses @techdocs/cli
```

**Comment by ericis** [comment]
I also tried installing `pip install mkdocs` and `pip install mkdocs-techdocs-core` and then `yarn start -v --no-docker` and still got a 404 error:

```shell
$ yarn start -v --no-docker
yarn run v1.22.18
$ techdocs-cli serve -v --no-docker
info: Starting mkdocs server.
verbose: [mkdocs] INFO     -  Building documentation...
verbose: [mkdocs] INFO     -  Cleaning site directory
verbose: [mkdocs] INFO     -  Documentation built in 0.81 seconds
verbose: [mkdocs] INFO     -  [12:44:19] Watching paths for changes: 'C:\Users\255450\AppData\Local\Temp\docs_krleks4e', 'mkdocs.yml', 'docs'
verbose: [mkdocs] INFO     -  [12:44:19] Serving on http://127.0.0.1:8000/
info: [techdocs-preview-bundle] Running local version of Backstage at http://localhost:3000
info: Serving docs in Backstage at http://localhost:3000/docs/default/component/local/
Opening browser.
verbose: [mkdocs] WARNING  -  [12:44:25] "GET /techdocs/index.html HTTP/1.1" code 404
```

**Comment by ericis** [comment]
Possibly related to #10989 and the PR https://github.com/backstage/backstage/pull/10945

**Comment by ctrombley** [comment]
Possibly related to #11466

**Comment by garyniemen** [comment]
@camilaibs can you take a look at this

**Comment by ericis** [comment]
Closed in favor of #10362

**Comment by ericis** [comment]
[This was resolved.](https://github.com/backstage/backstage/issues/10362#issuecomment-1128717028)
